### PR TITLE
Changes before submitting proposal

### PR DESCRIPTION
**Pull request by hugolatendresse**

### CODE REVIEW

File: _config.yml
Comment (mhk197) [inline _config.yml:1]: Parallelizing Mixture of Experts Transformers in FlexFlow for Low-Latency Inference

File: index.md [hunk 1]
```diff
@@ -16,77 +16,118 @@ Jump to:
...
-| Dec. 9 - Dec. 15      | Complete final report                                                                                          | 
+TODO complete after reading https://www.cs.cmu.edu/afs/cs/academic/class/15418-f24/www/projects/project-proposal.pdf
+
+
+| Week                  | Task                                                                                                        | 
+|-----------------------|-------------------------------------------------------------------------------------------------------------|
+| Nov. 11 - Nov. 17     | Finalize the project proposal                                                                               | 
+| Nov. 11 - Nov. 17     | Familiarize ourselves with the FlexFlow repo                                                                | 
+| Nov. 11 - Nov. 17     | Go through FlexFlow's developer guide                                                                       | 
+| Nov. 11 - Nov. 17     | Meet with Gabriele                                                                                          |
```
Comment (mhk197) [inline index.md]: Meet with a member of Prof Jia's reserach team

**Review by mhk197 — CHANGES_REQUESTED** [review]
Not sure about this inter-expert vs intra-expert stuff. I mentioned it once to motivate thinking about different ways we can parallelize, but in the end they might be the same and just a concatenated matrix mult. 

Added some other comments but otherwise, based on what I saw w prev project proposals, this looks good!

**Comment by hugolatendresse** [comment]
> Not sure about this inter-expert vs intra-expert stuff. I mentioned it once to motivate thinking about different ways we can parallelize, but in the end they might be the same and just a concatenated matrix mult.
> 
> Added some other comments but otherwise, based on what I saw w prev project proposals, this looks good!

@mhk197 I had a convo with chatgpt regarding intra/inter experts. What I wrote is what I got from that

**Pull request merged**